### ISSUE TITLE
F100 Not Displaying for New Broswer User (Mindy, the new snippets person)

### ISSUE THREAD

**Issue by stsegovia** [mozmeao/snippets-service#1140]
**TL;DR:** Mindy, the new snippets team member can't see F100 snippets, even though she is a new Firefox user. 

Hi @glogiotatidis ,

I was just setting up Mindy Xu, our new snippets person. We were browsing Firefox on her new laptop, but we don't see snippets on her new tabs. 

We reviewed the Snippets Dev tools and none of the F100 snippets seem to appear. Perhaps 

She is on version 68 and does not see any F100 snippets appear in the new tab. This makes me wonder if there is an issue with F100 during the move. Could you check? Thank you!

<img width="633" alt="Screen Shot 2019-09-05 at 10 11 23 AM" src="https://user-images.githubusercontent.com/47153345/64365988-2144a100-cfca-11e9-8be9-fc52091aba8d.png">

<img width="1000" alt="Screen Shot 2019-09-05 at 10 11 55 AM" src="https://user-images.githubusercontent.com/47153345/64365999-243f9180-cfca-11e9-8d22-55686303097a.png">

**Comment by glogiotatidis** [comment]
@stsegovia thanks for the report.

A new Firefox install triggers on boarding. While on boarding runs snippets are not displayed. 

You say new laptop so this is a new install, and she will get onboarding snippets for a while. On-boarding is not something Snippets platform controls.  
 
If my above assumption is incorrect and this is not a new install, or she completed on boarding already and she still doesn't get F100 snippets then the question is does she see other snippets?

You can also check F100 metrics via your dashboards to see if we get impressions. I checked in telemetry for job 21165 (snippet 11103) and I see impressions.

**Comment by stsegovia** [comment]
Ok. That makes sense. I just needed your 2nd opinion. I guess it's probably that she isn't getting the onboarding snippets. We'll wait and see if she gets them.
